.gitignore

```diff
@@ -3,3 +3,4 @@ npm-debug.log
 coverage*
 .tern-port
 v8.log
+package-lock.json
```

Comment:
package-lock.json should not be ignored 

https://docs.npmjs.com/files/package-lock.json

Reply:
did i do that? :D then revert